Rakefile

```diff
@@ -7,4 +7,4 @@ Osra::Application.load_tasks
 
 require 'coveralls/rake/task'
 Coveralls::RakeTask.new
-task :test_with_coveralls => [:spec, :cucumber, 'coveralls:push']
+task :test_with_coveralls => [:spec, :cucumber_hq, :cucumber_aa, 'coveralls:push']
```

Comment:
If people can give their feedback on #269, we can decide how best to handle this.